packages/x-components/src/x-modules/search/components/sort-picker-list.vue

```diff
@@ -12,7 +12,7 @@
       :class="[cssClasses, buttonClass]"
       data-test="sort-picker-button"
       :events="event"
-      :aria-pressed="item === selectedSort"
+      :aria-pressed="(item === selectedSort).toString()"
```

Comment:
I think now we have to refactor the test. We were expecting the attribute to be missing and now it is present but with false value